### PR TITLE
[11.0][MIG] partner_aging

### PR DESCRIPTION
**Pull request by murtuzasaleh**
#161

### CODE REVIEW

**Review by max3903 — APPROVED** [review]

**Review by dreispt — APPROVED** [review]
Code review - LGTM

**Review by dreispt — CHANGES_REQUESTED** [review]
Please check failing tests.

**Comment by max3903** [comment]
@dreispt Failing test is related to another module.

**Review by smangukiya — APPROVED** [review]
Code review LGTM

File: partner_aging/wizards/res_partner_aging_supplier.xml [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        <field name="name">Supplier Aging</field>
+        <field name="type">ir.actions.act_window</field>
+        <field name="res_model">res.partner.aging.date</field>
+        <field name="view_type">form</field>
+        <field name="view_mode">form</field>
+        <field name="view_id" ref="view_supplier_aging_date"/>
+        <field name="target">new</field>
+    </record>
+
+    <!-- Supplier Aging Menuitem -->
```
Comment (Swapnesh-SerpentCS) [inline partner_aging/wizards/res_partner_aging_supplier.xml]: Menu Item

File: partner_aging/wizards/res_partner_aging_supplier.xml [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+        </field>
+    </record>
+
+    <!-- Supplier Aging Tree -->
+    <record model="ir.ui.view" id="view_supplier_aging_ad_tree">
+        <field name="name">supplier.aging.ad.tree</field>
+        <field name="model">res.partner.aging.supplier</field>
+        <field name="type">tree</field>
+        <field name="arch" type="xml">
+            <tree create="false" delete ="false" string="Supplier Aging Date wise" colors="red:max_days_overdue &gt; 0 and total &gt; 0;">
```
Comment (Swapnesh-SerpentCS) [inline partner_aging/wizards/res_partner_aging_supplier.xml]: colors tag is depricated
Use  decoration-danger instaed

**Comment by max3903** [comment]
@murtuzasaleh Can you fix @Swapnesh-SerpentCS comments please?

**Comment by Swapnesh-SerpentCS** [comment]
I will fix this soon.

**Comment by Swapnesh-SerpentCS** [comment]
Hello @pedrobaeza 
Is it necessary to fix E501 in such case where SQL is too long?
https://travis-ci.org/OCA/account-payment/jobs/544608031#L586


Thanks in Advance.

**Comment by pedrobaeza** [comment]
Yes of course, you can use `"""` for enclosing long texts for example.

**Pull request merged**